common/src/main/java/com/microsoft/identity/common/internal/eststelemetry/EstsTelemetry.java

```diff
@@ -321,17 +363,63 @@ String getCurrentTelemetryHeaderString() {
             return null;
         }
 
-        return currentTelemetry.getCompleteTelemetryHeaderString();
+        return currentTelemetry.getCompleteHeaderString();
     }
 
-    String getLastTelemetryHeaderString() {
-        RequestTelemetry lastTelemetry = loadLastRequestTelemetryFromCache();
-
-        if (lastTelemetry == null) {
+    private String getLastTelemetryHeaderString() {
+        if (mLastRequestTelemetryCache == null) {
             return null;
         }
 
-        return lastTelemetry.getCompleteTelemetryHeaderString();
+        LastRequestTelemetry lastRequestTelemetry = (LastRequestTelemetry) mLastRequestTelemetryCache.getRequestTelemetryFromCache();
```

Comment:
nit : `final` here and everywhere else in the method for local variables

Reply:
Addressed